psql-users.sh

```diff
@@ -1,2 +1,3 @@
 psql -U postgres -c "create user test with superuser password 'test';"
-psql -U postgres -c "create database automated_test with owner test;"
\ No newline at end of file
+psql -U postgres -c "create database automated_test with owner test;"
+psql -U postgres -c "create database big_int_test with owner test;"
```

Comment:
Why do we need the `big_int_test` database?

Reply:
I was wondering that too. Presumably the automated_test database should have the big int columns.

Reply:
I might misunderstood the `Acceptance Criteria` in the ticket. :thinking: 

>  Use gdc-ng-models and gdcdatamodel to create tables in an empty (test) database. Confirm all ID fields are bigints rather than integers.

Reply:
I think it's fine to keep using the one database to test this. It should be created as a new database each test run.

Reply:
:+1: 
I think the database will still be there, but all the tables are dropped at the end of each testing session.
In the case of tests erroring-out before the clean up, we can always drop the tables manually and start fresh again.

Reply:
I can speak to the (confusing) acceptance criteria since I wrote them. 😓 The idea was to use the tools in those respective repositories to set up a new database in a dev cluster, not within the scope of the unit tests.

Reply:
...or you can use them to set up a new database locally.